modules/github-repository-webhook/variables.tf

```
@@ -38,3 +38,15 @@ variable "webhook_secret" {
   type        = string
   default     = ""
 }
+
+variable "webhook_insecure_ssl" {
+  description = "Disable verify SSL certificates when delivering payloads"
+  type        = bool
+  default     = false
+}
+
+variable "webhook_content_type" {
+  description = "Webhook Content Type"
+  type        = string
+  default     = "application/json"
```

Looking at the [GitHub terraform provider docs](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_webhook#content_type), valid values here would be `json` or `form`, which can be validated. 
```suggestion
  default     = "json"
  validation {                                                                                                                                                                        
    condition     = contains(["json", "form"], var.webhook_content_type)                                                                                                              
    error_message = "Valid values are either `json` or `form`"                                                                                                                        
  }
```